packages/swapper/src/swappers/RelaySwapper/constant.ts

```diff
@@ -58,13 +58,18 @@ export const relayChainIdToChainId = invert(chainIdToRelayChainId)
 
 export const DEFAULT_RELAY_EVM_TOKEN_ADDRESS = zeroAddress
 export const RELAY_BTC_TOKEN_ADDRESS = 'bc1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqmql8k8'
+export const RELAY_SOLANA_TOKEN_ADDRESS = '11111111111111111111111111111111'
 
 export const DEFAULT_RELAY_EVM_USER_ADDRESS = '0x000000000000000000000000000000000000dead'
 export const DEFAULT_RELAY_BTC_USER_ADDRESS = 'bc1q4vxn43l44h30nkluqfxd9eckf45vr2awz38lwa'
+export const DEFAULT_RELAY_SOLANA_USER_ADDRESS = 'CbKGgVKLJFb8bBrf58DnAkdryX6ubewVytn7X957YwNr'
 
 export const RELAY_SUPPORTED_CHAIN_IDS: SupportedChainIds = {
   sell: relaySupportedChainIds,
   buy: relaySupportedChainIds,
 }
 
 export const MAXIMUM_SUPPORTED_RELAY_STEPS = 2
+
+// Relay UTXO balance check requires 1 confirmation or the quote fetching will fail
+export const MINIMUM_RELAY_CONFIRMATIONS_UTXO = 1
```

Comment:
ser

Reply:
Fucking mergefix 😆 

Thrown our of the window in [da24f87](https://github.com/shapeshift/web/pull/9253/commits/da24f87f9ab751c8b45c87a3b50e173b4cf2735b)